src/app/pages/BuySovPage/components/BuyForm/index.tsx

```diff
@@ -115,18 +115,25 @@ export function BuyForm() {
               onChange={value => setAmount(value)}
               placeholder="0.0000"
               rightElement={<AssetRenderer asset={Asset.RBTC} />}
+              dataAttribute="buySovPage-enterAmount-field"
             />
             <Slippage>
-              <AvailableBalance asset={Asset.RBTC} />
+              <AvailableBalance
+                asset={Asset.RBTC}
+                dataAttribute="buySovPage-availableBalance-field"
+              />
             </Slippage>
-            <AmountButton onChange={changeAmount} />
+            <AmountButton
+              onChange={changeAmount}
+              dataAttribute="buySovPage-amount-"
+            />
           </FieldGroup>
 
           <ArrowDown />
 
           <FieldGroup label={t(s.fields.receive)} labelColor="#e8e8e8">
             <Dummy className="tw-flex tw-justify-between tw-items-center">
-              <div>
+              <div data-action-id="buySovPage-youReceive-amount-display">
```

Comment:
This and the other action keys seems very arbitrary to me…
There should be some sort of consistent pattern, similar to BEM.
pageName-componentName[-elementName] for example.

Reply:
@JoeStorm99

Reply:
Ok, please feel to use whatever you think is appropriate.

Reply:
I already told you, you need to think about the naming of the attributes. My job as a developer is to help you by adding those attributes to elements. So take your time, try to find appropriate naming, adjust the story description and validate it with us in the Light chat. Then, after it's validated, go through the other stories as well and adjust naming.